mlflow/models/evaluation/base.py

```diff
@@ -937,6 +937,8 @@ def evaluate(
           precision, recall, f1, etc. for multiclass classification models
           (default: ``'weighted'``). For binary classification and regression models, this
           parameter will be ignored.
+        - **sample_weights**: Weights for each sample to apply when computing model performance
```

Comment:
@jimmyxu-db I opted for a plural argument since the weights are almost always plural. `sample_weight` is a bit of a misnomer,  even though sklearn uses it